views/voicemails-rows.html

```diff
@@ -24,16 +24,16 @@
 			<div class="bottom-line">{{toFriendlyDate timestamp 'time'}}</div>
 		</td>
 		{{#if formatted.showCallerIDName}}
-			<td class="select-line no-padding" data-filter-value="{{from}} {{formatted.from}} {{ formatted.callerIDName }}">
+			<td class="select-line no-padding" data-filter-value="{{from}} {{formatted.from}} {{ formatted.callerIDName }}" data-sort-value="{{from}} {{formatted.from}} {{ formatted.callerIDName }}" >
```

Comment:
Sorting is different than filtering, we allow users to search (filter) the rows by every relevant fields available to them (the `from` prop might be in a log file if they are support people for example).

For sorting, we usually only expose fields that are visible to the user (except for timestamps and specific UI elements).
```suggestion
			<td class="select-line no-padding" data-filter-value="{{from}} {{formatted.from}} {{ formatted.callerIDName }}" data-sort-value="{{formatted.from}} {{ formatted.callerIDName }}" >
```